src/content/en/updates/_shared/devtools-research.md

```diff
@@ -0,0 +1,4 @@
+<aside class="note">
+  <p>Interested in helping improve DevTools? Sign up to participate in <a href="https://google.qualtrics.com/jfe/form/SV_3NMIMtX0F2zkakR?reserved=1&utm_source=Website%20feature&Q_Language=en&utm_medium=own_web&utm_campaign=Q4&productTag=chrm&campaignDate=November2020&referral_code=UXbl384838">Google User Research here</a>.
```

Comment:
Should this be a `target="_blank" rel="noopener"`?

Reply:
Updated